src/Styleguide/Components/Lightbox/Lightbox.tsx

```diff
@@ -0,0 +1,286 @@
+import { color, space } from "@artsy/palette"
+import FadeTransition from "Components/Animation/FadeTransition"
+import { bind, once, throttle } from "lodash"
+import React from "react"
+import ReactDOM from "react-dom"
+import styled from "styled-components"
+import { Box } from "Styleguide/Elements/Box"
+import { Flex } from "Styleguide/Elements/Flex"
+import { CloseButton } from "./CloseButton"
+import { Slider, SliderProps } from "./LightboxSlider"
+
+const KEYBOARD_EVENT = "keyup"
+const ZOOM_PER_CLICK = 1.4
+const HIDE_ZOOM_SLIDER_AFTER = 2500
+
+const DeepZoomContainer = styled.div`
+  position: fixed !important;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1000;
+  background-color: ${color("black100")};
+`
+
+export interface LightboxProps {
+  deepZoom: {
+    Image: {
+      xmlns: string
+      Url: string
+      Format: string
+      Overlap: number
+      TileSize: number
+      Size: {
+        Width: number
+        Height: number
+      }
+    }
+  }
+}
+
+export interface LightboxState {
+  shown: boolean
+  element: Element
+  viewer: any
+  deepZoomRef: any
+  slider: SliderProps
+  showZoomSlider: boolean
+  promisedDragon: Promise<any>
+  activityTimer?: NodeJS.Timer
```

Comment:
`NodeJS.Timer` isn't the right type here... it should be the browser timer. I was looking for it, but I've already lost so much time on this work that I decided this was good enough. Let me know if you know the correct type.

Reply:
I’m not sure what you mean by a ‘timer’. Do you mean a ‘timer ID’, such as those created by `setTimeout()`? If so, what I like to do is e.g. type out `setTimeout` in vscode and then grab the typing from either the intellisense popup or command-click on the API to go to the right ambient declaration file.